www/js/types/diaryTypes.ts

```diff
@@ -26,6 +24,21 @@ type ConfirmedPlace = {
       as a string (e.g. 'walk', 'drove_alone') */
     [k: `${string}confirm`]: string;
   };
+  exit_fmt_time: string;
+  exit_ts: number;
+  exit_local_dt: LocalDt;
+  starting_trip: ObjectId;
+};
+
+export type TripTransition = {
+  currstate: string;
+  transition: string;
+  ts: number;
+};
+
+export type LocationCoord = {
+  type: string; // e.x., "Point"
+  coordinates: [number, number] | Array<[number, number]>;
```

Comment:
What's the difference between `[number, number]` and `Array<[number, number]>`?
Why do you have to put both?